SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+The latest released version of Daml is supported with security updates.
+You can find the latest version at:
+https://docs.daml.com/support/release-notes.html
+
+## Reporting a Vulnerability
+
+If you suspect that your issue has a security impact, do not use the issue tracker.
+In that case, please follow our responsible disclosure policy:
+https://www.digitalasset.com/responsible-disclosure
```

Comment:
This confused me slightly because "the issue tracker" hasn't been mentioned before now. But I suppose in most cases people will get to this policy via the issue tracker, so in that sense it's been mentioned. :) 

Up to you. If you do want to make it a bit clearer, you could swap this paragraph with the next paragraph and change the heading to "Reporting an Issue or Vulnerability"

Reply:
I understand that; I'm just saying it hasn't been mentioned earlier on this page so it kind of came out of nowhere. Not a big deal.